test/test.baseexternalclient.ts

```diff
@@ -1307,6 +1529,55 @@ describe('BaseExternalAccountClient', () => {
         });
         scopes.forEach(scope => scope.done());
       });
+
+      it('should not throw if workforce user project is provided but client auth is not', async () => {
```

Comment:
Suggest renaming: should still pass workforce user project when no client auth is used